getting-started/setup-multi-node-auth.md

```diff
@@ -292,7 +393,36 @@ access nodes have keys and the user has a certificate.
 7. On all data nodes, add `USAGE` privileges on the `timescaledb_fdw` foreign data 
 wrapper (and any other objects, as necessary).
 
+###$ 5. Attach the data nodes to the access node
+
+Once the nodes are properly configured, you can continue following the [multi-node setup][attach_multi_node_nodes].
+
+#### 6. Setting up additional roles [](multi-node-auth-certificate-roles)
+
+Allowing new roles to use the certificate to authenticate is simply a matter of
+adding them to the certificate role.  Aside from that, the process of adding new
+users should be the same as for [trust authentication](multi-node-auth-trust-roles).
```

Comment:
Does the reference to trust authentication mean that it should be a common description of adding a user?